### DISCUSSION TITLE
GNU Nano Duplicating Characters On WSL (Under very special circumstances)

### DISCUSSION THREAD

**Discussion by just1a-person** [microsoft/WSL#6498] [General] [answered]
See https://github.com/cli/cli/issues/424

**Comment by AjkayAlan** [comment, marked as answer]
FYI, I have closed cli/cli#424, as I am no longer seeing the issue any more. My environment specifics that now result in things working well:

Distro: Ubuntu 20.04
GH CLI: V1.7.0
Nano: 4.8
Windows Terminal: 1.6.10571.0
